src/NzbDrone.Core.Test/MediaFiles/EpisodeImport/Specifications/FreeSpaceSpecificationFixture.cs

```diff
@@ -68,7 +68,7 @@ public void should_reject_when_there_isnt_enough_disk_space()
         }
 
         [Test]
-        public void should_reject_when_there_isnt_enough_space_for_file_plus_100mb_padding()
+        public void should_reject_when_there_isnt_enough_space_for_file_plus_min_free_space()
```

Comment:
This test fails because the default mocked result is `0`. and the 100MB file fits in the 150MB space.